### PR TITLE
Feature/reflexion merge into develop/

### PR DESCRIPTION
**Pull request by ThomasProg**
Reflexion added to GPGame and GPEditor.

### CODE REVIEW

File: engine/third_party/include/Refureku/Kodgen/CodeGen/DefaultGeneratedCodeTemplate.h [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/**
+*	Copyright (c) 2020 Julien SOYSOUVANH - All Rights Reserved
+*
+*	This file is part of the Kodgen library project which is released under the MIT License.
+*	See the README.md file for full license details.
+*/
+
+#pragma once
+
+#include "Kodgen/CodeGen/GeneratedCodeTemplate.h
```
Comment (badsami) [inline engine/third_party/include/Refureku/Kodgen/CodeGen/DefaultGeneratedCodeTemplate.h:10]: Missing `"` here
```suggestion
#include "Kodgen/CodeGen/GeneratedCodeTemplate.h"
```

**Review by badsami — COMMENTED** [review]
Launching the project in debug mode returns these errors:
image

`RefurekuBin/` only contains `RefurekuSettings.toml`. `RefurekuGenerator.exe` and `vswhere.exe` are listed in GitHub, but appear `empty`. They do not exist in the repository

There is a `Bar.hpp` file in `projects/GPGame/include/`
There is a `Foo.hpp` in `engine/include/Engine/`

**EDIT**: something went wrong when I pulled your branch. My bad, everything works!

**Comment by Renardjojo** [comment]
Ok for me. I didn't see any compil error

**Review by Renardjojo — APPROVED** [review]

**Review by badsami — APPROVED** [review]
Appart from the missing `"` in `engine/third_party/include/Refureku/Kodgen/CodeGen/DefaultGeneratedCodeTemplate.h` (which is not from you anyway and doesn't seem to be causing any error, strangely), everything works fine!

**Pull request merged**